library/models/employee.py

```diff
@@ -13,22 +13,23 @@ class User(BaseModel):
 
     @staticmethod
     def from_neo4j(record: Record):
-        print("Element id", record['result'].element_id, type(record['result']['element_id']))
-        return User(id=record['result'].element_id, employee_id=record['result']['employee_id'], 
-                    name=record['result']['name'], email=record['result']['email'])
+        print("Element id", record['result'].element_id, type(record['result'].element_id))
+        print("Employe from", record)
+        return User(id=record['result'].element_id, employee_id=record['result'].get('employee_id'), 
+                    name=record['result'].get('name'), email=record['result']['email'])
```

Comment:
Adding a name to the token response to allow bilven to render it without another call